src/main/java/com/onarandombox/MultiverseNetherPortals/listeners/MVNPEntityListener.java

```diff
@@ -200,4 +229,12 @@ public void onEntityPortalEnter(EntityPortalEnterEvent event) {
             this.plugin.log(Level.FINE, "Player '" + p.getName() + "' was allowed to go to '" + toWorld.getCBWorld().getName() + "' because enforceaccess is off.");
         }
     }
-}
+
+    public void onEntityPortalExit(EntityPortalExitEvent event) {
+        if (event.getEntity() instanceof Player) {
+            Player p = (Player) event.getEntity();
+            this.eventRecord.get(PortalType.ENDER).remove(p);
+            this.eventRecord.get(PortalType.NETHER).remove(p);
+        }
+    }
+}
```

Comment:
Is this suppose to be annotated with `@EventHandler`? Also, let's keep new lines at the end of files as a general rule.

Reply:
Good catch!